### PR TITLE
Update change-nameservers-at-any-domain-registrar.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to add Custom records

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4780

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 1]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
 
 2. On the **Domains** page, select the domain and then choose **DNS Records**.
 
-3. Under **Manage DNS**, select **Custom Records**, and then choose **New custom record**.
+3. Under **Manage DNS**, select **Custom Records**, and then choose **New custom record** and enter the following.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
3. Under **Manage DNS**, select **Custom Records**, and then choose **New custom record** and enter the following:
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 2]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
 
-4. Select the type of DNS record you want to add, and type the information for the new record.
-
-5. Select **Save**.
+   - For **DNS type** enter: **A (Address)**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   - For **DNS type**, enter **A (Address)**.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 3]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
+    
+   - For **Host name or Alias**, type the following: **@**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   - For **Host name or Alias**, enter **@**.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 4]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
+    
+   - For **IP Address**, type the static IP address for your website where it's currently hosted (for example, 172.16.140.1).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   - For **IP Address**, enter the static IP address for your website where it's currently hosted (for example, 172.16.140.1). 
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 5]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
-4. Select the type of DNS record you want to add, and type the information for the new record.
-
-5. Select **Save**.
+   - For **DNS type** enter: **A (Address)**
+    
+   - For **Host name or Alias**, type the following: **@**
+    
+   - For **IP Address**, type the static IP address for your website where it's currently hosted (for example, 172.16.140.1). 
+    
+   This must be a  *static*  IP address for the website, not a  *dynamic*  IP address. Check with site where your website is hosted to make sure you can get a static IP address for your public website.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   This must be a  *static*  IP address for the website, not a  *dynamic*  IP address. Check with the site where your website is hosted to make sure you can get a static IP address for your public website. 
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 6]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
+    
+   This must be a  *static*  IP address for the website, not a  *dynamic*  IP address. Check with site where your website is hosted to make sure you can get a static IP address for your public website. 
+    
+5. Select **Save**. 
+    
+In addition, you can create a CNAME record to help customers find your website.
+  
+1. Select **+ New custom record** and enter the following: 
+    
+   - For **DNS type** enter: **CNAME (Alias)**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   - For **DNS type**, enter **CNAME (Alias)**.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 7]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
+    
+   - For **Host name or Alias**, type the following: **www**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   - For **Host name or Alias**, enter **www**.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 8]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
+    
+   - For **Points to address**, type the fully qualified domain name (FQDN) for your website (for example, contoso.com).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
   - For **Points to address**, enter the fully qualified domain name (FQDN) for your website (for example, contoso.com). 
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 9]
```diff
@@ -161,11 +161,31 @@ For example, here are some additional steps that might be required for email and
...
+    
+   - For **DNS type** enter: **CNAME (Alias)**
+    
+   - For **Host name or Alias**, type the following: **www**
+    
+   - For **Points to address**, type the fully qualified domain name (FQDN) for your website (for example, contoso.com). 
+    
+2. Select **Save**. 
+  
+When the NS records have been updated to point to Microsoft, your domain is all set up. Email will be routed to Microsoft, and traffic to your website address will continue to go to your current website host.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
When the nameserver records have been updated to point to Microsoft, your domain is all set up. Email will be routed to Microsoft, and traffic to your website address will continue to go to your current website host.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@scotvidican Please merge this PR. Thanks

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please replace everything from line 165 through 187 with the following:

````
2. On the **Domains** page, select a domain.
3. On the domain details page, select the **DNS records** tab.
4. Select **Add record**.
5. In the **Add a custom DNS record** pane, from the **Type** dropdown list, select **A (Address)**.
6. In the **Host name or Alias** box, type **@**.
7. In the **IP Address** box, type the static IP address for the website where it's currently hosted. For example, 172.16.140.1.
> [!IMPORTANT]
>
> This must be a *static* IP address for the website, not a *dynamic* IP address. To make sure you can get a static IP address for your public website, check with the site that hosts your website.
8. If you want to change the TTL setting for the record, select a new length of time from the **TTL** dropdown list. Otherwise, continue to step 9.
9. Select **Save**.

In addition, you can create a CNAME record to help customers find your website.

1. Select **Add record**.
2. In the **Add a custom DNS record** pane, from the **Type** dropdown list, select **CNAME (Alias)**.
3. In the **Host name or Alias** box, type **www**.
4. In the **Points to address** box, type the fully qualified domain name (FQDN) for your website. For example, **contoso.com**.
5. If you want to change the TTL setting for the record, select a new length of time from the **TTL** dropdown list. Otherwise, continue to step 6.
6. Select **Save**.

After the nameserver records are updated to point to Microsoft, your domain setup is complete. Email is routed to Microsoft, and traffic to your website address continues to go to your current website host.
````

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review @cmcatee-MSFT The changes have been implemented

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 10]
```diff
@@ -159,13 +159,36 @@ For example, here are some additional steps that might be required for email and
 
 1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834818" target="_blank">Domains</a> page.
 
-2. On the **Domains** page, select the domain and then choose **DNS Records**.
+2. On the **Domains** page, select a domain.
 
-3. Under **Manage DNS**, select **Custom Records**, and then choose **New custom record**.
+3. On the domain details page, select the **DNS records** tab.
+ 
+4. Select **Add** record.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
4. Select **Add record**.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 11]
```diff
@@ -159,13 +159,36 @@ For example, here are some additional steps that might be required for email and
...
+
+5. In the **Add a custom DNS record pane**, from the **Type** dropdown list, select **A (Address)**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
5. In the **Add a custom DNS record** pane, from the **Type** dropdown list, select **A (Address)**.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 12]
```diff
@@ -159,13 +159,36 @@ For example, here are some additional steps that might be required for email and
...
+
+6. In the **Host name or Alias** box, type **@**.
+
+7. In the **IP Address** box, type the static IP address for the website where it's currently hosted. For example, 172.16.140.1.
+    
+> [!IMPORTANT]
+>  This must be a static IP address for the website, not a dynamic IP address. To make sure you can get a static IP address for your public website, check with the site that hosts your website.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
>  This must be a _static_ IP address for the website, not a _dynamic_ IP address. To make sure you can get a static IP address for your public website, check with the site that hosts your website.
```

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 13]
```diff
@@ -159,13 +159,36 @@ For example, here are some additional steps that might be required for email and
...
+In addition, you can create a CNAME record to help customers find your website.
+  
+1.	Select **Add record**.
 
-4. Select the type of DNS record you want to add, and type the information for the new record.
+3.	In the **Add a custom DNS record** pane, from the **Type** dropdown list, select **CNAME (Alias)**.
+4.	In the **Host name or Alias** box, type **www**.
+5.	In the **Points to address** box, type the fully qualified domain name (FQDN) for your website. For example, **contoso.com**.
+6.	If you want to change the TTL setting for the record, select a new length of time from the **TTL** dropdown list. Otherwise, continue to step 6.
+7.	Select Save.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
7.	Select **Save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**